src/geo/src/GeoPMTFactoryBase.cc

```diff
@@ -120,6 +120,33 @@ namespace RAT {
         int    light_cone       = table->GetI("light_cone");
         bool   lightcones = false;
         if( light_cone == 1 ){ lightcones = true; G4cout << "Light cones are added!! \n "; }
+        //material properties
+        G4Material* light_cone_material = G4Material::GetMaterial("TiO2");
+	try { light_cone_material = G4Material::GetMaterial( table->GetS("light_cone_material") ); }
+	catch (DBNotFoundError &e) { }
+        //surface properties
+        G4SurfaceProperty* light_cone_surface = Materials::optical_surface["TiO2"];
```

Comment:
Also switch this to tyvek

Reply:
i have switched the default light_cone_material and light_cone_surface to "aluminum" since that is the most accurate at this time.